src/pages/markets/static/content/_derived.tsx

```diff
@@ -8,33 +8,34 @@ import FriendlySupport from 'images/svg/markets/friendly-support-new.svg'
 import ResponsiveWebsite from 'images/svg/markets/responsive-website-new.svg'
 import PractiseIcon from 'images/svg/markets/aim-new.svg'
 import TradeIcon from 'images/svg/markets/trade-new.svg'
+import { TString } from 'types/generics'
+
+export type SimpleStepContentElement = {
```

Comment:
you can create a global type for this because you are repeating this type in different files